### ISSUE TITLE
[Outlining] Lambdas should be folded into next line

### ISSUE THREAD

**Issue by vasily-kirichenko** [fsprojects/VisualFSharpPowerTools#1204]
```fsharp
Job.iterateServer (createSlidingTimes(), createTotalTimes()) (fun (slidingTimes, totalTimes) ->
            (addTimeCh |>>? fun (stage, time) -> 
                let q = slidingTimes.[stage]
                q.Enqueue time
                if q.Count > timeCountToKeep then q.Dequeue() |> ignore
                let totalTimes = totalTimes |> Map.add stage (Time.Mappend totalTimes.[stage] time)
                slidingTimes, totalTimes) <|>?
```
![image](https://cloud.githubusercontent.com/assets/873919/10667744/8e88cd0c-78e1-11e5-9b51-1ac2062cca10.png)
![image](https://cloud.githubusercontent.com/assets/873919/10667751/99f2f514-78e1-11e5-97d6-a458df91e7e0.png)

I'd prefer the collapsed `[...]` to be placed on the next line. Currently it erases code structure.

**Comment by cloudRoutine** [comment]
Putting lambdas below puts us back at this issue
![lambda_below](https://cloud.githubusercontent.com/assets/680051/10679550/6c148512-78e7-11e5-9c24-eb0116b28e7e.gif)

**Comment by cloudRoutine** [comment]
fixed in #1206
